docs/serving/autoscaling/autoscaling-concepts.md

```diff
@@ -0,0 +1,135 @@
+
+Knative Serving supports the implementation of Knative Pod Autoscaler (KPA) and Kubernetes' Horizontal Pod Autoscaler (HPA). The features and limitations of each of these Autoscalers are listed below.
+
+**IMPORTANT:** If you want to use Kubernetes Horizontal Pod Autoscaler (HPA), you must install it after you install [Knative Serving](../../install/any-kubernetes-cluster.md#optional-serving-extensions).
+
+### Knative Pod Autoscaler (KPA)
+
+* Part of the Knative Serving core and enabled by default once Knative Serving is installed.
+* Supports scale to zero functionality.
+* Does not support CPU-based autoscaling.
+
+### Horizontal Pod Autoscaler (HPA)
+
+* Not part of the Knative Serving core, and must be enabled after [Knative Serving installation](../../install/any-kubernetes-cluster.md#optional-serving-extensions).
+* Does not support scale to zero functionality.
+* Supports CPU-based autoscaling.
+
+### Configuring the Autoscaler implementation
+
+The type of Autoscaler implementation (KPA or HPA) can be configured by using the `class` annotation.
+
+* **Global settings key:** `pod-autoscaler-class`
+* **Per-revision annotation key:** `autoscaling.knative.dev/class`
+* **Possible values:** `"kpa.autoscaling.knative.dev"` or `"hpa.autoscaling.knative.dev"`
+* **Default:** `"kpa.autoscaling.knative.dev"`
+
+**Example:**
+{{< tabs name="class" default="Per Revision" >}}
+{{% tab name="Per Revision" %}}
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-go
+  namespace: default
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/class: "kpa.autoscaling.knative.dev"
+    spec:
+      containers:
+        - image: gcr.io/knative-samples/helloworld-go
+```
+{{< /tab >}}
+{{% tab name="Global (ConfigMap)" %}}
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+ name: config-autoscaler
+ namespace: knative-serving
+data:
+ pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+```
+{{< /tab >}}
+{{% tab name="Global (Operator)" %}}
+```yaml
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    autoscaler:
+      pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Global versus per-revision settings
+
+Configuring for autoscaling in Knative can be set using either global or per-revision settings.
+
+1. If no per-revision autoscaling settings are specified, the global settings will be used.
+1. If per-revision settings are specified, these will override the global settings when both types of settings exist.
+
+### Global settings
+
+Global settings for autoscaling are configured using the `config-autoscaler` ConfigMap. If you installed Knative Serving using the Operator, you can set global configuration settings in the `spec.config.autoscaler` ConfigMap, located in the `KnativeServing` custom resource (CR).
+
+#### Example of the default Kubernetes ConfigMap
```

Comment:
```suggestion
#### Example of the default autoscaling ConfigMap
```

"Kubernetes ConfigMap" makes me think that this is doing something to Kubernetes, rather than simply stored in Kubernetes.